com.unity.netcode.gameobjects/Runtime/Messaging/MessagingSystem.cs

```diff
@@ -8,6 +8,11 @@
 
 namespace Unity.Netcode
 {
+    internal class HandlerNotRegisteredException : SystemException
```

Comment:
You defined this but then didn't use it?

suggest adding something like this on line 368:

```
            if (handler == null)
            {
                Debug.LogException(new HandlerNotRegisteredException(header.MessageType.ToString()));
            }
            else
            {
```